src/Elyon.Fastly.Api.Domain/Services/IEmailSenderService.cs

```diff
@@ -36,5 +36,7 @@ public interface IEmailSenderService : IBaseService
             DateTime poolingDate, ICollection<int> shifts);
 
         Task SendInfoSessionFollowUpEmail(string receiver, string messageContent, string confirmationToken);
+
+        Task SendOnboardingEmail(string receiver, IEnumerable<string> ccReceivers, int organizationTypeId, Dictionary<string, string> parameters);
```

Comment:
All Task methods should end with Async. Please also rename the others in this service